### PR TITLE
Improve split step

### PR DESCRIPTION
**Pull request by StefanoGuerrini**
Reviewers: @GetFeedback/platform-java @singhanee

### CODE REVIEW

File: docs/features/transformers.md [hunk 1]
```diff
@@ -56,16 +56,24 @@ Pulls a field out of a complex value and replaces the entire value with the extr
       field: id
 ```
 
-## Split Value
-Split an array into values.
+## Split array
+Split an array into multiple records.
```
Comment (vadzim-kazak) [inline docs/features/transformers.md:60]: Could you please explain in more detail in documentation what is expected behavior of this transformer?
"jmesPath" contains a path to the array. But I didn't get what is the purpose of the "to" configuration property.

File: docs/features/transformers.md [hunk 2]
```diff
@@ -56,16 +56,24 @@ Pulls a field out of a complex value and replaces the entire value with the extr
...
       config:
         jmesPath: "value.fooArray"
+        to: "value.foo" # Optional field
 ```
 
+### Optional configurations
+
+| name | default | description                              |
+|------|---------|------------------------------------------|
+| to   | ""      | define where to put the object extracted |
```
Comment (StefanoGuerrini) [inline docs/features/transformers.md:74]: this is the explanation @vadzim-kazak 
Is it not clear?

File: docs/features/transformers.md (hunk 1 above)
Comment (StefanoGuerrini) [reply docs/features/transformers.md:60]: I'll put more info on the general behavior

File: docs/features/transformers.md (hunk 2 above)
Comment (vadzim-kazak) [reply docs/features/transformers.md:118]: yep, for me without KaHPP knowledge it is not clear. I understand this transformer as for any object with a nested array we are going to split this array to separate records. Does it mean that if "to" property is not empty a new object is going to be created and array value stored in it under "to" path? If yes, I guess we can describe this behavior under description column.

Comment (StefanoGuerrini) [reply docs/features/transformers.md:118]: Good idea!

Comment (StefanoGuerrini) [reply docs/features/transformers.md]: Updated

File: docs/features/transformers.md (hunk 1 above)
Comment (vadzim-kazak) [reply docs/features/transformers.md:60]: Much better now, thanks.

**Review by vadzim-kazak — APPROVED** [review]

**Pull request merged**